WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletInfoViewModel.cs

```diff
@@ -37,17 +37,17 @@ public WalletInfoViewModel(WalletViewModelBase walletViewModelBase)
 			var secret = PasswordHelper.GetMasterExtKey(wallet.KeyManager, wallet.Kitchen.SaltSoup(), out _);
 
 			ExtendedMasterPrivateKey = secret.GetWif(network).ToWif();
-			ExtendedAccountPrivateKey = secret.Derive(wallet.KeyManager.AccountKeyPath).GetWif(network).ToWif();
+			ExtendedAccountPrivateKey = secret.Derive(wallet.KeyManager.SegwitAccountKeyPath).GetWif(network).ToWif();
```

Comment:
no

Reply:
No because it is out of the scope of this PR. There is an endless list of things that need to be changed in the UI after taproot:

* update the output descriptors
* remove the gaps report 
* remove the customs keypaths
* remove the incomplete code for notifying after the mingaplimit is increased
* etc. 

There is another important set of changes needed for coinjoins and i suspect that another set of changes will be needed in the coinselection algoritm to make sure we don't ruin everything. Things like verifying the correct fee for a new type of scripts and .... 

btw, i have just realized the transaction builder doesn't know how to sign transaction spending taproots. ufff.

I want to focus on having this PR.